config.go

```diff
@@ -461,6 +495,13 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
+	if err := cfg.parsePubKeys(activeNetParams.Params); err != nil {
+		str := "%s: Failed to parse Pubkeys"
+		err := fmt.Errorf(str+": %v", funcName, err)
```

Comment:
Please avoid string concatenation here. You can instead use

```
err := fmt.Errorf("%s: Failed to parse Pubkeys: %v", funcName, err)
```

Reply:
Yes sir!

I also realized the check should be made after checking whether keys are set at all, so moved it down a bit.